extensions/amp-analytics/0.1/visibility-model.js

```diff
@@ -196,11 +213,15 @@ export class VisibilityModel {
     this.unsubscribe_.length = 0;
     this.eventResolver_ = null;
     // TODO(jonkeller): Investigate why dispose() can be called twice,
-    // necessitating this "if"
-    if (this.onTriggerObservable_) {
-      this.onTriggerObservable_.removeAll();
-      this.onTriggerObservable_ = null;
+    // necessitating this "if", and the same "if" elsewhere in this file.
+    if (!this.onTriggerObservable_) {
+      dev().warn(TAG_,
```

Comment:
Its ok not to warn here as we already expect that it may not exist so leave the code as is

Reply:
I disagree. See the "before" on https://github.com/ampproject/amphtml/pull/12308/files. It being unexpectedly null here is what started this whole thing.

Reply:
Ah I didn't realize you had changed it in your previous PR... I thought that was the original code.